packages/editor-ui/src/components/mixins/targetBlank.ts

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue';
+
+export const targetBlank = Vue.extend({
+	methods: {
+		addTargetBlank(html: string) {
+			return html.includes('a href')
```

Comment:
this is fragile. if an extra space is added by mistake it will not work. can we find a better approach?